### PR TITLE
Fix feedback button saving

### PR DESCRIPTION
**Pull request by darkruby501**
Makes it so requesting feedback saves the draft. I wanted it that it would save without taking you out of the editor, but I haven't figured out our forms enough to do that.

### CODE REVIEW

File: packages/lesswrong/components/posts/PostSubmit.tsx [hunk 1]
```diff
@@ -84,17 +84,19 @@ const PostSubmit = ({
...
-                    'requested-feedback', 
-                    {title: document.title, _id: document._id, url: "https://www.lesswrong.com/" + document._id}
-                  )
+                  if (!!document.title) {
+                    updateCurrentValues({draft: true});
+                    // eslint-disable-next-line
+                    (window as any).Intercom(
+                      'trackEvent',
+                      'requested-feedback',
+                      {title: document.title, _id: document._id, url: "https://www.lesswrong.com/posts/" + document._id}
```
Comment (jimrandomh) [inline packages/lesswrong/components/posts/PostSubmit.tsx:97]: This should use `getSiteUrl()` (for compatibility with Alignment Forum, EA forum, etc).

**Review by jimrandomh — APPROVED** [review]

File: packages/lesswrong/components/posts/PostSubmit.tsx (hunk 1 above)
Comment (Discordius) [reply packages/lesswrong/components/posts/PostSubmit.tsx:97]: Yeah, does seem like this whole button should probably be hidden on EA Forum. Though showing it on AIAF is probably fine.

Comment (darkruby501) [reply packages/lesswrong/components/posts/PostSubmit.tsx:97]: Makes sense.

Comment (darkruby501) [reply packages/lesswrong/components/posts/PostSubmit.tsx:97]: New PR is up

**Pull request merged**